CRT.c

```diff
@@ -1260,8 +1260,9 @@ static void print_backtrace(void) {
 #elif defined(HAVE_EXECINFO_H)
    void* backtraceArray[256];
 
-   size_t size = backtrace(backtraceArray, ARRAYSIZE(backtraceArray));
-   backtrace_symbols_fd(backtraceArray, size, STDERR_FILENO);
+   int nptrs = backtrace(backtraceArray, ARRAYSIZE(backtraceArray));
+   assert(nptrs >= 0);
```

Comment:
If you check this assert, do proper error handling instead by writing "Backtrace failed" or alike.

Reply:
@BenBE  What do you mean? `backtrace(3)` never documents it can return a negative value, so I won't know what error it can produce. `nptrs == 0` might be a good case to handle by letting it print something rather than nothing.